docker/README.md

```diff
@@ -13,17 +14,19 @@
 The following step builds a docker image(`spark/spark-web-ui:latest`) using `Dockerfile` and `pom.xml` file. 
 
 ```shell
-git clone https://github.com/Hyper-Mesh/spark-history-server.git
-cd docker
-docker build -t <docker-user-id>/spark-web-ui:latest . 
+git clone https://github.com/krishnadasmallya/spark-history-server.git
```

Comment:
This is pointing to your personal repo. Can this be changed to git clone https://github.com/Hyper-Mesh/spark-history-server.git?